code/game/machinery/computer/prisoner.dm

```diff
@@ -161,12 +164,12 @@
 					implant.warn_cooldown = world.time + IMPLANT_WARN_COOLDOWN
 					if(implant.imp_in)
 						var/mob/living/carbon/implantee = implant.imp_in
-						var/warning = copytext(sanitize(text2num(answer)), 1, MAX_MESSAGE_LEN)
+						var/warning = copytext(sanitize(answer), 1, MAX_MESSAGE_LEN)
 						to_chat(implantee, "<span class='boldnotice'>Your skull vibrates violently as a loud announcement is broadcasted to you: '[warning]'</span>")
 				if("set_points")
 					if(isnull(text2num(answer)))
 						return
-					inserted_id.goal = max(text2num(answer), 0)
+					inserted_id.goal = clamp(text2num(answer), 0, 50000)
```

Comment:
```suggestion
					inserted_id.goal = clamp(text2num(answer), 0, 9000)
```
this limit is 5000 not 50000 according to SoP, that equates to about 33 minutes roughly. Perhaps it should be changed to 9000, which is closer to 60 minutes

Reply:
This is a good point. I've set it to 20,000 which will cap the amount at around 2 hours and added comments for my justification for this cap.